aptos-move/block-executor/src/proptest_types/baseline.rs

```diff
@@ -230,154 +230,166 @@ impl<K: Debug + Hash + Clone + Eq> BaselineOutput<K> {
         }
     }
 
-    // Used for testing, hence the function asserts the correctness conditions within
-    // itself to be easily traceable in case of an error.
-    pub(crate) fn assert_output<E: Debug>(
-        &self,
-        results: &BlockExecutionResult<BlockOutput<MockOutput<K, E>>, usize>,
-    ) {
+    fn assert_success<E: Debug>(&self, block_output: &BlockOutput<MockOutput<K, E>>) {
```

Comment:
git is showing changes here because of indentation, this function is basically the same code as it was before, it's just extracted to be able to be used in both assert_output and assert_parallel_output